### ISSUE TITLE
Total in networth is different than total in the account list

### ISSUE THREAD

**Issue by andreva7** [clovett/MyMoney.Net#61]
I see an approximately 0.01% (1 in 10,000) difference between the totals reported when I look at the account list vs the sum of the table in networth.

Do you see that as well? 
All the asset values are exactly the same, so it likely is related to stock value calculation.
It seems a little high for it to be a rounding error, but maybe.

**Comment by clovett** [comment]
Yeah, I'm seeing the same thing.  One reason for this can be the "Balance" shown in the account list is computed using whatever is in the Security Price field (for performance reasons) whereas the Networth report now uses the real downloaded StockQuote History, so if for any reason these two numbers are out of sync you will see some differences here.  

Another reason for differences is just rounding errors, as the Networth report is summing up things in different groups of transactions (by SecurityType and by TaxFree or not and so on).

The other difference can come from how it treats "closed" accounts.  The balance in the Accounts List ignores them I think whereas the Networth Report does not ignore them (if you do a report not based on today's date).  So there was a bug that I just found in this area.  If you run the default networth report for "today" then it uses Security.Price, but if you run it for yesterday it uses the StockQuote History, but it wasn't correctly loading all the StockQuote Histories, so you would see a big difference in the report.  I just pushed some fixes for this so the reports for yesterday, today, and tomorrow should now be relatively similar, but still not identical to the Account list balance.

**Comment by clovett** [comment]
I pushed some fixes that got things a lot closer on my machine:
- Portfolio was using DateTime.Today instead of DateTime.Now, the account balance is using DateTime.Now.
- Account balance was removing anything with UnitsRemaining < 1, Portfolio was not, so I added those back into account balance.
- Note that account balance on a brokerage account is market value of stocks remaining and cash balance, whereas on Portfolio report the investment cash is separated out into separate row.  This is still the case.
- I added cash balance to the single account portfolio report.

With these fixes I'm able to see now that my Fidelity ESPP account balance matches exactly with it's portfolio report.
